man/hbv.Rd

```diff
@@ -0,0 +1,135 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/hbv.R
+\name{hbv}
+\alias{hbv}
+\alias{hbv.sim}
+\alias{hbvrouting}
+\alias{hbvrouting.sim}
+\alias{hbv.ranges}
+\alias{hbvrouting.ranges}
+\title{HBV rainfall-runoff model}
+\usage{
+hbv.sim(
+  DATA,
+  tt,
+  cfmax,
+  sfcf,
+  cwh,
+  cfr,
+  fc,
+  lp,
+  beta,
+  k0,
+  k1,
+  k2,
+  uzl,
+  perc,
+  return_state = FALSE
+)
+
+hbvrouting.sim(U, maxbas, epsilon = hydromad.getOption("sim.epsilon"))
+
+hbv.ranges()
+
+hbvrouting.ranges()
+}
+\arguments{
+\item{DATA}{A time-series like object with columns P (precipitation in mm),
+E (potential evapotranspiration in mm), T (average air temperature in ºC).}
+
+\item{tt}{Threshold temperature for snow and snow melt in degrees Celsius.}
+
+\item{cfmax}{Degree-day factor for snow melt (mm/(ºC.day)).}
+
+\item{sfcf}{Snowfall correction factor. Amount of precipitation below
+threshold temperature that should be rainfall instead of snow.}
+
+\item{cwh}{Liquid water holding capacity of the snowpack.}
+
+\item{cfr}{Refreezing coefficient for water in the snowpack.}
+
+\item{fc}{Maximum amount of soil moisture storage (mm).}
+
+\item{lp}{Threshold for reduction of evaporation. Limit for potential
+evapotranspiration.}
+
+\item{beta}{Shape coefficient in soil routine.}
+
+\item{k0}{Recession coefficient (quick runoff).}
+
+\item{k1}{Recession coefficient (upper groundwater storage).}
+
+\item{k2}{Recession coefficient (lower groundwater storage).}
+
+\item{uzl}{Threshold for quick runoff for k0 outflow (mm).}
+
+\item{perc}{Maximum percolation from upper to lower groundwater storage.}
+
+\item{return_state}{Whether to return the state variables.}
+
+\item{U}{Effective rainfall series}
+
+\item{maxbas}{Routing, length of triangular weighting function.}
+
+\item{epsilon}{Values smaller than this in the output will be set to zero.}
+}
+\description{
+An implementation of the HBV rainfall-runoff model.
+}
+\details{
+This implementation of this HBV model closely follows the
+description of HBV light by Seibert and Vis, 2009. Daily average temperature
+data is required for the snow routine. Daily potential evapotranspiration
+(PET) data is required as the routine to calculate daily PET using long term
+mean PET and daily temperature is not included.
+
+The timeseries of simulated streamflow (U). If return state is set
+to true, the state variables of the model are also returned. These include:
+snowpack (sp), water content of snowpack (wc), soil moisture (sm), actual
+evapotranspiration (ETa), upper groundwater storage (uz), lower groundwater
+storage (lz).
+}
```

Comment:
Could you please add a comment about the source/reasoning behind values for default parameters and parameter ranges?